src/map/battle.cpp

```diff
@@ -6828,6 +6853,12 @@ struct Damage battle_calc_attack(int attack_type,struct block_list *bl,struct bl
 	}
 	else // Some skills like Weaponry Research will cause damage even if attack is dodged
 		d.dmg_lv = ATK_DEF;
+
+	struct map_session_data *sd = BL_CAST(BL_PC, bl);
+
+	if (sd && d.damage + d.damage2 > 1)
```

Comment:
You changed the condition to trigger Vanish damage from "greater than 0 damage" to "greater than 1 damage". Is there any bug if it's the former? I need it in my server.